client/lib/plans/constants.js

```diff
@@ -282,19 +326,27 @@ export const PLANS_LIST = {
 		getStoreSlug: () => PLAN_JETPACK_PERSONAL,
 		availableFor: ( plan ) => includes( [ PLAN_JETPACK_FREE ], plan ),
 		getPathSlug: () => 'jetpack-personal',
-		getDescription: () => i18n.translate(
-			'Essentials for every site. The most affordable solution to keep your' +
-			' personal or small business site backed up and spam-free.'
+		getDescription: ( abtest ) => abtest && abtest( 'jetpackNewDescriptions' ) === 'showNew'
+			? i18n.translate( 'Security essentials for every site.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Security essentials for every site' )` *ES Score: 21*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).